client/src/routes/Pages/components/Footer/Footer.js

```diff
@@ -6,21 +6,21 @@ const Footer = () => (
   <footer>
     <div>
       <a href="https://m.do.co/c/ddb2a965377c" target="_blank" rel="noreferrer">
```

Comment:
As a matter of fact, the change was required here but leaving this all the other elements were changed :confused: